tests/integrations/raftstore/test_stats.rs

```diff
@@ -123,3 +128,196 @@ fn test_store_heartbeat_report_hotspots() {
     fail::remove("mock_tick_interval");
     fail::remove("mock_hotspot_threshold");
 }
+
+type Query = dyn Fn(Context, Cluster<ServerCluster>, TikvClient, u64, u64, Vec<u8>);
+
+#[test]
+fn test_query_stats() {
```

Comment:
The testcases seem only including the `read` case?

Reply:
There are test cases about `put` in line253 and line296